tools/ci/stage.sh

```diff
@@ -106,6 +106,7 @@ flink-connectors/flink-connector-nifi,\
 flink-connectors/flink-connector-rabbitmq,\
 flink-connectors/flink-connector-twitter,\
 flink-connectors/flink-connector-kinesis,\
+flink-connectors/flink-connector-aws-kinesis-data-streams,\
```

Comment:
Nit: next commit?

Reply:
same as above, but this time the ci build fails. 😢